.gitignore

```diff
@@ -1,2 +1,3 @@
 # Ignore stuff from the builds that get generated.
 build/
+docs/
```

Comment:
We _should_ commit the `docs/` markdown to the repo so we can automatically generate it and integrate it

Reply:
Isn't it more of a build artifact? Every PR has the potential to make the docs out of date.

That said, we could have the pizza-action generate docs whenever a PR goes up.

Reply:
Anyway, that said, I've added the first iteration of the CLI docs to the PR.

Reply:
> Isn't it more of a build artifact

Yes, it's essentially a build artifact. But we also need living docs in this repository. And, ultimately, `pizza docs` should serve the purpose of making it as easy and seamless for us to keep those up to date with the code.

I'd imagine the next step here is to have another step on the release action that runs `pizza docs`, updates the docs under the `docs/` directory in the repo, and then commits it.